src/gui/optionsdlg.h

```diff
@@ -60,6 +60,8 @@ namespace Ui
 class OptionsDialog: public QDialog
 {
     Q_OBJECT
+    using ThisType = OptionsDialog;
```

Comment:
I guess you still like the name `ThisType` more... so be it.
For me, a type means "data type", the traditional define you'll find on wikipedia... so its not so intuitive to me as `ThisClass` (literally).